src/main/java/net/starschema/clouddb/jdbc/DMDResultSet.java

```diff
@@ -485,8 +485,7 @@ public DMDResultSet(Object[] objects, String[] colnames, DMDResultSetType type)
   public void close() throws SQLException {
     super.close();
     this.Colnames = null;
-  }
-  ;
+  };
```

Comment:
Apparently different versions disagree about what to do about extraneous semicolons after function definitions.  I should just remove that instead of reformatting it...